release/models/optical-transport/openconfig-transport-types.yang

```diff
@@ -1109,6 +1115,16 @@ module openconfig-transport-types {
     description "Ethernet compliance code: 10GBASE_SR";
   }
 
+  identity ETH_25GBASE_LR {
+    base ETHERNET_PMD_TYPE;
+    description "Ethernet compliance code: 25GBASE_LR";
+  }
+
+  identity ETH_25GBASE_SR {
+    base ETHERNET_PMD_TYPE;
+    description "Ethernet compliance code: 25GBASE_SR";
+  }
```

Comment:
Please add a reference to the desired specification for this PMD.  I think in this case it's probably `802.3by-2016` ?

Reply:
Could we reference the 802.3-2022 standard in which case this would be 802.3 clause 112.

Reply:
I added a reference to 802.3by-2022 (CL112).